docs/API/Logging.md

```diff
@@ -39,93 +39,93 @@ These advantages are mainly for BetonQuest, but it is also very useful for 3rd p
+        but it will still cache the instances for the same topic or without a topic.
+    ---
+ 
+After you now have a `BetonQuestLoggerFactory` instance, you can create a `BetonQuestLogger` instance using one of the following methods:
 
-
-!!! warning "Get the logger in your JavaPlugin class"
-    The methods described above don't work for your plugin's main class (or any other class that extends `Plugin`). 
-    Create the logger instance in the `onEnable()` method instead.
+!!! note ""
 
     === "Without topic"
-        ````java linenums="1"
-        public final class BetonQuestAddon extends JavaPlugin {
-    
-            private static BetonQuestLogger log;
+        !!! abstract ""
+            This is a the standard way to create a BetonQuestLogger instance.
+            
+            ````java linenums="1"
+            final BetonQuestLogger logger = loggerFactory.create(MyClass.class);
+            ````
     
-            @Override
-            public void onEnable() {
-                log = BetonQuestLogger.create(this);
-        ````
-
     === "With topic"
-        ````java linenums="1"
-        public final class BetonQuestAddon extends JavaPlugin {
+        !!! abstract ""
+            This is useful if you want to give your log messages a prefix like `(Database)`.
+            Mainly _PascalCase_ should be used for topics and they should be short and meaningful to the user. 
+            
+            ````java linenums="1"
+            final BetonQuestLogger logger = loggerFactory.create(MyClass.class, "MyCustomTopic");
+            ````
     
-            private static BetonQuestLogger log;
+    !!! warning "Get the logger in your JavaPlugin class"
+        The methods described above don't work for your plugin's main class (or any other class that extends `Plugin`). 
+        Create the logger instance in the `onEnable()` method instead like this:
     
-            @Override
-            public void onEnable() {
-                log = BetonQuestLogger.create(this, "MyCustomTopic");
-        ````
+        === "Without topic"
+            ````java linenums="1"
+            public final class BetonQuestAddon extends JavaPlugin {
+        
+                private static BetonQuestLogger log;
+        
+                @Override
+                public void onEnable() {
+                    log = BetonQuestLogger.create(this);
+            ````
+    
+        === "With topic"
+            ````java linenums="1"
+            public final class BetonQuestAddon extends JavaPlugin {
+        
+                private static BetonQuestLogger log;
+        
+                @Override
+                public void onEnable() {
+                    log = BetonQuestLogger.create(this, "MyCustomTopic");
+            ````
 
 ## Using the BetonQuestLogger
 A BetonQuestLogger will be available as the variable `LOG` once you [obtained a BetonQuestLogger instance](#obtaining-a-betonquestlogger-instance). 
```

Comment:
This part still assumes a static `BetonQuestLogger` instance stored in the field `LOG`.

Remove this line and change the following line to:
```markdown
The BetonQuestLogger has a bunch of methods for all use cases. Its JavaDocs explain when and how to use these.
```
A suggestion was no option as this text is out of scope of the changes in GitHub.